web/index.pug

```diff
@@ -7,6 +6,12 @@ block append variables
 
 block maincontent
 	.page.content
+		.art 
+		span.icon
+ 			svg(width="24px", height="24px")
```

Comment:
- the indentation here is broken, being a mixture of tabs and spaces (we need to use tabs-only)